ldb/operational_test.go

```diff
@@ -16,22 +17,15 @@ import (
 
 	"github.com/btcsuite/btcdb"
 	"github.com/btcsuite/btcnet"
+	"github.com/btcsuite/btcscript"
 	"github.com/btcsuite/btcutil"
 	"github.com/btcsuite/btcwire"
+	"golang.org/x/crypto/ripemd160"
 )
 
 var network = btcwire.MainNet
 
-func TestOperational(t *testing.T) {
-	testOperationalMode(t)
-}
-
-func testOperationalMode(t *testing.T) {
-	// simplified basic operation is:
-	// 1) fetch block from remote server
-	// 2) look up all txin (except coinbase in db)
-	// 3) insert block
-
+func setUpTestDb(t *testing.T) (btcdb.Db, []*btcutil.Block, string, string, error) {
```

Comment:
Great suggestion, changed to pack it all into a new struct that includes a closure to be called to clean-up the initialized state.